code/game/objects/items/toys.dm

```diff
@@ -1292,4 +1293,35 @@
 			user.put_in_hands(A)
 		else
 			new /obj/item/clothing/gloves/anchor_arms(get_turf(src.loc))
-		qdel(src)
\ No newline at end of file
+		qdel(src)
+
+/obj/item/toy/balloon/decoy
+	name = "inflatable decoy"
+	desc = "Use this to fool your enemies into thinking you're a balloon!"
+	icon_state = "decoy_balloon_deflated"
+	w_class = W_CLASS_TINY
+	col = null
+	inflated_type = /obj/item/toy/balloon/inflated/decoy
+	volume = 120	//liters
+
+/obj/item/toy/balloon/decoy/inflate(mob/user, datum/gas_mixture/G)
+	var/obj/item/toy/balloon/inflated/decoy/D = ..()
+	if(!istype(D))
+		return
+	user.drop_item(D, force_drop = 1)
+	D.name = user.name
+	D.desc = user.desc
```

Comment:
Of course.

I don't mean copy over the code for item, just copy the text itself.